### ISSUE TITLE
Purchase HP DL360 Gen9 servers for Dublin

### ISSUE THREAD

**Issue by pnorman** [openstreetmap/operations#525]
- [ ] Get specs
- [ ] Get quotes
- [ ] Sort out DB4 shipping
- [ ] Purchase

-----

Tentative specifications are
2x E5-2695v4 (or 2690v4 or 2680v4, depending on current costs
128GB DDR4-2400
HPE Ethernet 10Gb 2-port 561FLR-T Adapter
HPE Smart Array P440ar/2GB FBWC (Does this need the HP Smart Storage Battery?)
3x HP 1.2TB SSD SATA drives
2x HPE 500W supply
HPE easy install rails kit

est. cost 3.9k EUR

**Comment by pnorman** [comment]
I realized that the specs listed are different than agreed on at the meeting. I'll update them based on the meeting.

**Comment by tomhughes** [comment]
We're going to duplicate forum, piwik, gps tile, munin, management and the the entire foundation server?

When was that decided, and how exactly do you imagine it working? Virtually none of that is set up to be able to be multihomed!

**Comment by pnorman** [comment]
My understanding was we wanted to move everything hosted in the UK to being hosted in Dublin. Some of them are old enough we should have replaced them last year if not for COVID, and the newest of them is a G7.

**Comment by tomhughes** [comment]
Oh sorry I was think those were Amsterdam machines that you were trying to duplicate in Dublin.

**Comment by pnorman** [comment]
Should we order all 10 servers at once, or split it up over time? Ideally everything would be staggered, but I don't think we can delay a second batch enough for it to matter, so I'm in favour of getting them all at once for the greatest standardization.

**Comment by Firefishy** [comment]
Yes, we should include the HP Smart Storage Battery with the HPE Smart Array P440ar

**Comment by pnorman** [comment]
Pro-forma sent to treasurer.

**Comment by pnorman** [comment]
Shipping in progress - I need to file an Equinix ticket

**Comment by Firefishy** [comment]
All servers racked and up and running. Services still to be setup, but separate tickets.
